spec/support/retry_helpers.rb

```diff
@@ -57,7 +58,7 @@ def reload_url_until_status_code(
     }
 
     retry_while_false(reload_options) do
-      code = HTTParty.head(url).code
+      code = HTTParty.head(url, follow_redirects: follow_redirects).code
```

Comment:
Do you happen to know if we hit problems if this is just set to false rather than being configurable? 
I feel like since this method is in the context of status_codes already that we can just tell it to expect a redirect (as I believe we do in the test for this). Might keep things a bit simpler being non-configurable.